graylog2-web-interface/src/legacy/Rickshaw.Graph.Graylog2Selector.js

```diff
@@ -116,17 +116,24 @@ const Graylog2Selector = Rickshaw.Class.create({
     }, false);
 
         // Stop at chart boundaries.
-    if (graph.dataDomain()[0] === position.xMin) {
-      graph.window.xMin = undefined;
-    }
-    if (graph.dataDomain()[1] === position.xMax) {
-      graph.window.xMax = undefined;
+    console.log('running', graph.series);
```

Comment:
There is a stray `console.log` in here. :smile:

Reply:
Thanks! ✅